test/python/pulse/test_macros.py

```diff
@@ -91,6 +92,91 @@ def test_fail_measure(self):
         with self.assertRaises(PulseError):
             macros.measure(qubits=[0], inst_map=self.inst_map)
 
+    def test_measure_v2(self):
+        """Test macro - measure with backendV2."""
+        sched = macros.measure(qubits=[0], backend=self.backend_v2).filter(
```

Comment:
Can you add another test with multiple qubits? Also it's better not to filter output (i.e. testing full schedule equality), because some future PR may break this function by injecting invalid instruction. This should be captured by this test.

Reply:
I added a multiple test at https://github.com/Qiskit/qiskit-terra/pull/9987/commits/381a8e65d8a4586269ec59ca74a86a01a98661db.